sofana/R3BSofFrsAnalysis.cxx

```diff
@@ -303,11 +303,17 @@ void R3BSofFrsAnalysis::Exec(Option_t* option)
             i_s8cave = i;
         }
         if (isnan(tof) || tof < 0)
+        {
             continue;
-        beta.push_back(fPathLength[i] / (tof + fTofOffset[i]));
+        }
+        else
+        {
+            beta.push_back(fPathLength[i] / (tof + fTofOffset[i]));
```

Comment:
Using push_back would not fit with the later analysis. Can it be as follow instead?:
`beta.at(i) = fPathLength[i] / (tof + fTofOffset[i]);`